app/models/docker_container_wizard_states/image.rb

```diff
@@ -4,8 +4,32 @@ class Image < ActiveRecord::Base
     belongs_to :wizard_state, :class_name => DockerContainerWizardState,
                               :foreign_key => :docker_container_wizard_state_id
     delegate :compute_resource_id, :to => :wizard_state
+    delegate :compute_resource, :to => :wizard_state
 
     validates :tag,             :presence => true
     validates :repository_name, :presence => true
+    validate :validate_image_exists
+
+    def name
+      "#{repository_name}:#{tag}"
+    end
+
+    def registry_name
+      @registry_name ||= if registry_id
+        DockerRegistry.find(registry_id).fetch(:name, _('Unnamed Registry'))
+      else
+        'Docker Hub'
+      end
+    end
+
+    def validate_image_exists
+      unless compute_resource.exist? name
+        error_msg = _("Container image %{image_name} could not be found on %{registry}.") % {
```

Comment:
Yes, I missed putting it as a inline help for the field. Gonna do that right away.